process/throttle/antiflood/p2pBlackListProcessor.go

```diff
@@ -0,0 +1,108 @@
+package antiflood
+
+import (
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+type p2pBlackListProcessor struct {
+	cacher                     storage.Cacher
+	blacklistHandler           process.BlackListHandler
+	thresholdNumReceivedFlood  uint32
+	thresholdSizeReceivedFlood uint64
+	numFloodingRounds          uint32
+}
+
+// NewP2pBlackListProcessor creates a new instance of p2pQuotaBlacklistProcessor able to determine
+// a flooding peer and mark it accordingly
+func NewP2pBlackListProcessor(
```

Comment:
P2P

Reply:
I deliberately avoided P2P because it will have double upper case letters: "P2**PB**lackList"

Reply:
that is no problem. P2P should be.

Reply:
Done. Refactored whole project. Now we follow the pattern xxxP2PXxxx. (thus lip2p's packages do not follow this rule :D )